pages/create-study.js

```diff
@@ -60,18 +63,40 @@ export default function CreateStudy() {
         error={!isValidParticipantLimit}
         helperText={isValidParticipantLimit ? '' : 'Must be between 1 and 500'}
       />
-      <TextField
-        id="outlined-basic"
-        label="Input profile"
-        variant="outlined"
-        type="number"
-        value={inputProfile}
-        onChange={(e) => setInputProfile(e.target.value)}
-        sx={{ mb: 4 }}
-        inputProps={{ min: 0, max: 16 }}
-        error={!isValidInputProfile}
-        helperText={isValidInputProfile ? '' : 'Must be between 0 and 16'}
-      />
+      <FormControl style={{minWidth: 120}}>
+        <InputLabel id="outlined-basic">Input Profile</InputLabel>
+        <Select
+          labelId="outlined-basic-label"
+          id="outlined-basic"
+          type="number"
+          defaultValue={0}

```

Comment:
I think the following props can be removed:
labelId, id, type, defaultValue, inputProps, error, helperText (don't need to validate since the user can't input a value not in the select)

Also remove `const isValidInputProfile = inputProfile >= 0 && inputProfile <= 16` above